chains/manager.go

```diff
@@ -183,9 +184,10 @@ type ManagerConfig struct {
 	Router                      router.Router              // Routes incoming messages to the appropriate chain
 	Net                         network.Network            // Sends consensus messages to other validators
 	Validators                  validators.Manager         // Validators validating on this chain
-	NodeID                      ids.NodeID                 // The ID of this node
-	NetworkID                   uint32                     // ID of the network this node is connected to
-	Server                      server.Server              // Handles HTTP API calls
+	LiteSyncPrimaryNetwork      bool
```

Comment:
nit: put next to `NetworkID`

Reply:
moved - lmk if that was where you were thinking